src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      class="search-form-big"
+      data-qa="big"
+    >
+      <label 
+        class="search-big"
+        for="label-for-input-big"
+      >
+        <input
+          class="input"
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          id="label-for-input-big"
+        >
+        <img
+        class="icon"
+        src="./images/Search.svg" 
+        alt="search-icon"
+        >
+      </label>
+    </form>
+    <form
```

Comment:
add an empty line between _multiline_ siblings
```suggestion
    </form>
    
    <form
```